setup.cfg

```diff
@@ -32,7 +32,7 @@ install_requires =
     craft-parts
     craft-providers
     overrides
-    spdx-license-list
+    spdx-lookup
```

Comment:
This project has [one release on pypi](https://pypi.org/project/spdx-lookup/#files) from 2020 and is basically just a wrapper around [another module](https://pypi.org/project/spdx/) also last updated in 2020.

It might be easier and more reliable to just create a list of license IDs ourselves with:
```python3
license_ids = [lic['licenseId'].lower() for lic in requests.get('https://github.com/spdx/license-list-data/raw/main/json/licenses.json').json()['licenses']]
```

or including the latest `licenses.json` in the wheel file at packaging time.

Reply:
I agree that the package isn't in great shape, but I tried to do the "quickest" thing that would un-break `main` for now. The other solutions would need a bit more effort (and I'm not sure we should add a runtime dependency to github.com), I just didn't want rockcraft to stay broken over the holidays ;)

I added #169 to review this early next year.

Reply:
I'm +1 for unbreaking main for the holidays.  

That package is so small and infrequently updated, that forking and maintaining may be an option :thinking: 
Or keeping a hardcopy of https://github.com/spdx/license-list-data/raw/main/json/licenses.json in this repo.

Reply:
The other option is to use SPDX's [`spdx-tools`](https://github.com/spdx/tools-python) package, but that has problems of its own (including also having an outdated package file).

I agree that making this a January problem is probably the best way to go, so I'll put opinions in #169